boltdesignsystem/bolt
# DS-303 Button Twig update

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-303](https://pegadigitalit.atlassian.net/browse/DS-303)

## Summary

Update Button Twig conventions.

## Details

This updates Button to use `{% set this = init(schema) %}`. Also, we remove the `base_class` variable, switch from double to single quotes, and update any other Twig standards that were out of date.

## How to test

- Review files changed
- Visually compare Button demos and other components that use Button with demos on master

> Note: you'll see in Jest snapshots that default prop values are no longer output to `<bolt-button>`.
